connect/src/test/java/com/stripe/android/connect/EmbeddedComponentManagerTest.kt

```diff
@@ -1,27 +1,47 @@
 package com.stripe.android.connect
 
+import android.Manifest
+import android.app.Application
+import androidx.activity.ComponentActivity
+import androidx.test.core.app.ApplicationProvider
+import junit.framework.TestCase.assertFalse
+import kotlinx.coroutines.Dispatchers
+import kotlinx.coroutines.async
+import kotlinx.coroutines.test.advanceUntilIdle
 import kotlinx.coroutines.test.runTest
+import kotlinx.coroutines.test.setMain
 import org.junit.Before
 import org.junit.Test
+import org.junit.runner.RunWith
 import org.mockito.Mockito.mock
 import org.mockito.Mockito.verify
 import org.mockito.kotlin.any
 import org.mockito.kotlin.whenever
+import org.robolectric.Robolectric
+import org.robolectric.RobolectricTestRunner
+import org.robolectric.Shadows.shadowOf
 import kotlin.test.assertEquals
+import kotlin.test.assertNotNull
 import kotlin.test.assertNull
+import kotlin.test.assertTrue
 
 @OptIn(PrivateBetaConnectSDK::class)
+@RunWith(RobolectricTestRunner::class)
 class EmbeddedComponentManagerTest {
 
     private lateinit var configuration: EmbeddedComponentManager.Configuration
     private lateinit var mockFetchClientSecretCallback: FetchClientSecretCallback
     private lateinit var embeddedComponentManager: EmbeddedComponentManager
+    private lateinit var testActivity: ComponentActivity
 
     @Before
     fun setup() {
+        Dispatchers.setMain(Dispatchers.Unconfined)
```

Comment:
Need to `resetMain()` in this test class, too

Reply:
Oops, I don't even need this dispatcher and forgot to remove it 😅